ldk/javascript/src/utils/network.mapper.ts

```diff
@@ -0,0 +1,99 @@
+import { TextDecoder } from 'text-encoding-shim';
```

Comment:
Would it be worthwhile to move this to `src/network/network.mapper.ts`? It's becoming apparent that we will need mappers for many aptitudes, and I thought putting individual mappers closer to them might make sense.

Reply:
I'm actually not sure what would be the most idiomatic way here. I saw applications with both structures. Do you know specifically why would you do one way vs another?